### DISCUSSION TITLE
LEDE 每次编译直接编译master吗

### DISCUSSION THREAD

**Discussion by potoo0** [coolsnowwolf/lede#11002] [Q&A / 快速问答] [answered]
版本为什么不打 tag 呀？比如 最近的这个 [linux-firmware: update to 20230310](https://github.com/coolsnowwolf/lede/commit/a73aec8f8d093022ce933e23846d1bc2c9e3dadd)

每次编译直接拿 master 的编译吗？

**Comment by WYC-2020** [comment, marked as answer]
因为打了没意义，你看这个更新频率就知道，谁都不能保证打的版本 是绝对稳定可靠，所以干脆还不如不打

**Reply by lysgwl** [reply]
不能保证，为什么官方还出release版本？
